### ISSUE TITLE
Reddit header doesn't follow navbar

### ISSUE THREAD

**Issue by aehlke** [andreamazz/AMScrollingNavbar#349]
**Describe the bug**
In the Reddit webview example, when the navbar collapses, the Reddit header floats with a gap above it.

**To Reproduce**
Simply open the included webview demo and collapse the navbar.

**Expected behavior**
This could be a Reddit bug, but I'm wondering if there's anything that could be done to resolve this so that the Reddit header follows the navbar as it collapses.

**Comment by tungdx** [comment]
I got the same issue when running on iOS 12.1.

**Comment by andreamazz** [comment]
Yeah, I noticed that. Not sure what to do about it, to be honest with you (besides changing the sample URL 😬 )
It's more of an issue with their frontend.

**Comment by tungdx** [comment]
Hi @andreamazz FYI, I tested on Safari and Chrome, it did not happen

**Comment by andreamazz** [comment]
I see. Still, no idea why it's not adapting to the viewport. Any idea is welcome.

**Comment by tungdx** [comment]
Anyway, thank you so much for sharing a very good library.
